INSTALL.md

```diff
@@ -119,12 +119,11 @@ sudo ldconfig
 
 ##### Compile toxcore
 
-Then clone this repo, generate makefile, and install `toxcore` system-wide:
+Then clone this repo, run `cmake`, and install `toxcore` system-wide:
 ```bash
 git clone https://github.com/irungentoo/toxcore.git
```

Comment:
It won't work on irungentoo/toxcore for now (and I don't know how long it'll take for that to happen).